docs/tutorials/ifci_px4_flight_controller.rst

```diff
@@ -0,0 +1,304 @@
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/QuadcopterModules.png
+    :align: center
+    :scale: 50
+    :alt: Modules on Quadcopter
+
+    Module Organization on Quadcopter
+
+First we will set the flight controller specific parameters. ``ARMING_BEHAVE`` can be left as the default “Use Motor Arm Behave”. ``DISARM_TRIGGER`` will be set to “Coast Motors”. This will turn the motor controllers off and allow the motors to spin freely. In ``TELEM_IDS_1`` 0, 1, 2, 3 will be selected to indicate that the flight controller should request telemetry from those module IDs (because this is a bitset parameter, the actual number that is set will be 15). ``VERTIQ_NUM_CVS`` will be set to 4. This means that your IFCI packet will be filled with 4 control signals and that the available control value indices will be 0, 1, 2, and 3. More can be read about this in the :ref:`IFCI documentation<controlling_ifci>`. After these parameters are set, the flight controller will need to be rebooted.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/px4_settings.png
+    :align: center
+    :scale: 50
+    :alt: PX4 Module Parameters
+
+    PX4 Module Parameters
+
+Next the module parameters will be set. First the ``TARGET_MODULE_ID`` parameter will be set to 0. The Module Params should be reloaded. ``The CONTROL_MODE`` parameter will be set to PWM. The ``MAX_VELOCITY`` and ``MAX_VOLTS`` parameters can be ignored because we are not controlling the motor with velocity or voltage mode. If ``CONTROL_MODE`` is set to one of those, the MAX\_ parameter that it corresponds to needs to be set appropriately. ``VERTIQ_FC_DIR`` should be set to 2D. All of these parameters will be the same for all of the modules. The last two parameters ``THROTTLE_CVI``, and ``VERTIQ_MOTOR_DIR`` will vary with each module. In our example ``THROTTLE_CVI`` will be set to match the module ID. Since we are currently setting module ID 0’s settings, we will set ``THROTTLE_CVI`` to 0. ``VERTIQ_MOTOR_DIR`` will be set to 2D Counter Clockwise to match the diagram of the quadcopter above. Ensure the ``VERTIQ_MOTOR_DIR`` parameter matches the direction shown in the diagram for each module ID. This is then repeated for each module ID by selecting a new ``TARGET_MODULE_ID``. The resulting parameters for each module are shown below. Blue boxes are the parameters that match between modules, red are the ones that vary, and yellow is the module ID.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/module_settings.png
+    :align: center
+    :scale: 50
+    :alt: All Modules Set Correctly
+
+    All Modules Set Correctly
+
+
+Actuator Setup
+==============
+
+Now the actuators must be set up in the actuator tab of QGroundControl. Navigate to the actuator tab of QGRoundControl and ensure that the Vertiq IO tab shows up.
```

Comment:
Reword this sentence. Also you have a QGRoundControl typo.

Now, we will configure PX4's actuator outputs to map with our IFCI outputs. Navigate to QGroundControl's actuator tab, and ensure that the ...

something like that...but you know...better

Reply:
Maybe fixed? Leaving unresolved